tests/js/tjshello.nim

```diff
@@ -1,4 +1,5 @@
 discard """
+  cmd: "nim $target $options --stackTrace:off --lineTrace:off $file"
```

Comment:
If you're wondering why this is here, fixing top level compile options resulted in stack traces actually getting enabled at top level which bumps this above 1000 characters. I didn't disable stack traces for system includes at first, which generated 20000 characters, but I fixed that and now it's just below 4000. So we have to explicitly disable it.